src/services/UserService.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiResponse } from 'apisauce';
+
+import api from 'config/api';
+import { User } from 'types/user';
+import { objectCamelToSnake } from 'utils/case';
+
+export async function signUp(user: User): Promise<ApiResponse<any>> {
+  const response = await api.post('/users', objectCamelToSnake(user));
```

Comment:
You don't need to use async/await just return the promise

Reply:
Has been changed :godmode: